src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/PushMetadataToBuildAssetRegistry.cs

```diff
@@ -413,6 +413,11 @@ private string GetAzDevRepository()
             return GetEnv("BUILD_REPOSITORY_URI");
         }
 
+        private string GetAzDevRepositoryName()
+        {
+            return GetEnv("BUILD_REPOSITORY_NAME");
```

Comment:
Does this not need handling for null / unset ?

Reply:
This is a built-in environment variable (similar to the URI version). If it's not set then it throws (see GetEnv)